app/models/preserved_copy.rb

```diff
@@ -63,4 +64,14 @@ def update_status(new_status)
   def matches_po_current_version?
     version == preserved_object.current_version
   end
+
+  # Input: Takes a Time object, String, or Nil
+  # Output: If String method will return a time object,
+  # if nil, method will return the start of the Epoch Time Object
+  # and if Time object, it will return the same Time object.
```

Comment:
I think the comments are largely unnecessary -- they read like the code itself, no?   What do you think about something like

```ruby
# Wrangles time object, string or nil into what is needed for SQL query
# Note that nil defaults to the start of the Epoch so only PreservedCopy objects without a last_version_audit timestamp will be processed
```

In some ways the above comments are in the code already, or more accurately belong in the README or ... somwhere else, perhaps?  That last one could be a `@param` comment for public methods taking date arg ...

Reply:
i think you're right. The comment is in the code already - and is not needed. Looking through the code you can tell that all of that is happening. I'll remove it :)